### PR TITLE
RTL fixes for state info

### PR DESCRIPTION
**Pull request by yosilevy**
## Proposed change

RTL fixes for state info

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/state-summary/state-card-alert.ts [hunk 1]
```diff
@@ -55,7 +55,9 @@ class StateCardAlert extends LitElement {
           align-items: center;
         }
         ha-entity-toggle {
-          margin: -4px -16px -4px 0;
+          margin: -4px;
```
Comment (piitaya) [inline src/state-summary/state-card-alert.ts:58]:
```suggestion
          margin: -4px -16px -4px 0;
```

File: src/state-summary/state-card-toggle.ts [hunk 2]
```diff
@@ -36,7 +36,9 @@ class StateCardToggle extends LitElement {
       haStyle,
       css`
         ha-entity-toggle {
-          margin: -4px -16px -4px 0;
+          margin: -4px;
```
Comment (piitaya) [inline src/state-summary/state-card-toggle.ts:39]:
```suggestion
          margin: -4px -16px -4px 0;
```

**Review by piitaya — COMMENTED** [review]
Can we keep the previous margin for browsers that doesn't support these properties?

**Comment by yosilevy** [comment]
@piitaya Done although I do think some people added code that didn't do this. I can look around and see if I find any.

**Review by piitaya — APPROVED** [review]

**Pull request merged**